doc/manual/core_device.rst

```diff
@@ -188,7 +196,7 @@ One useful application of the EBAZ4205 is controlling external devices like the
 | RESET (Unused)           | RESET               | N/A: Bit Controlled        |
 +--------------------------+---------------------+----------------------------+
 
-For a step-by-step guide, see the `EBAZ4205 and AD9834 setup guide <https://newell.github.io/projects/ebaz4205>`_.
+For a guide, see the `EBAZ4205 and AD9834 setup guide <https://newell.github.io/projects/ebaz4205>`_.
```

Comment:
Is it not step-by-step anymore?

Reply:
It never really was "step-by-step" to begin with so taking it out is being more accurate.